aps/utils/roxar/job.py

```diff
@@ -23,9 +23,11 @@
 from aps.utils.roxar.progress_bar import APSProgressBar
 from aps.utils.check_rms_interactive_or_batch import check_rms_execution_mode
 
+from fmu.tools.rms.copy_rms_param_to_ertbox_grid import check_grid_layout
```

Comment:
We should add [`flu.tool`](https://github.com/equinor/fmu-tools) as a dependency in `pyproject.toml`

Reply:
I will check which version fmu.tools will get when new komodo release will come.

Reply:
The current version of fmu.tools is 1.20.2 (to be released later today or tomorrow)